dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -156,11 +162,13 @@ public EventSearchParams map(EventOperationParams operationParams)
     return searchParams
         .setProgram(program)
         .setProgramStage(programStage)
-        .setOrgUnit(orgUnit)
+        .setOrgUnit(requestedOrgUnit)
```

Comment:
Why do we need to keep track of both the `requestedOrgUnit` and the `accessibleOrgUnit`?